pkg/reconciler/v1alpha1/taskrun/taskrun.go

```diff
@@ -186,29 +180,15 @@ func (c *Reconciler) reconcile(ctx context.Context, tr *v1alpha1.TaskRun) error
 
 	// taskrun has finished (as child build has finished and status is synced)
 	if len(tr.Status.Conditions) > 0 && tr.Status.Conditions[0].Status != corev1.ConditionUnknown {
-		logger.Infof("finished %s", tr.Name)
+		logger.Infof("Finished %s", tr.Name)
 		return nil
 	}
 
-	if !haveBuild {
-		// Get related task for taskrun
-		t, err := c.taskLister.Tasks(tr.Namespace).Get(tr.Spec.TaskRef.Name)
-		if err != nil {
-			return err
-		}
-
-		// make build obj from task buildspec
-		logger.Infof("make build: %s", tr.Name)
-		if b, err = c.makeBuild(t, tr); err != nil {
-			return fmt.Errorf("failed to create a build for taskrun: %v", err)
-		}
-	}
-
 	// sync build status with taskrun status
 	if len(b.Status.Conditions) > 0 {
-		logger.Infof("syncing taskrun conditions with build conditions %s", b.Status.Conditions[0])
+		logger.Infof("Syncing taskrun conditions with build conditions %s", b.Status.Conditions[0])
 	} else {
-		logger.Infof("syncing taskrun conditions with build conditions []")
+		logger.Infof("Syncing taskrun conditions with build conditions []")
```

Comment:
i think you're going to see a bunch of conflicts with #128 cuz I made some minor modifications to a bunch of these lines :weary: (changed the messages, used `c.Logger`) - sorry about this!

Reply:
i have a feeling yours will get in first XD